index.d.tmpl

```diff
@@ -43,46 +43,7 @@ declare namespace depcheck {
       [filePath: string]: any;
     };
   }
-
```

Comment:
This is not norma is it?

Reply:
It is. Instead of directly writing the list of parsers, detectors and specials, the index is now a template, and the `component.js` script is completing it with all the files found in appropriate directories, just the same way it is creating the `component.json` file.
If you are specifically talking about the empty line, each group (parser, detector and special) is written starting with an empty line. If you run `npm run component`, you will see that the generated `dist/index.d.ts` is the same as the previous `index.d.ts` (at least, it should be!)